notebooks/211-speech-to-text/211-speech-to-text.ipynb

```diff
@@ -0,0 +1,559 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "cf874fc4",
+   "metadata": {},
+   "source": [
+    "# Speech to Text with OpenVINO\n",
+    "\n",
+    "This notebook presents how to do speech to text recognition.\n",
+    "\n",
+    "For this tutorial [quartznet 15x5](https://docs.openvino.ai/2021.4/omz_models_model_quartznet_15x5_en.html) is used. QuartzNet model performs automatic speech recognition. QuartzNet’s design is based on the Jasper architecture, which is a convolutional model trained with Connectionist Temporal Classification (CTC) loss. Mentioned model can be found in [Open Model Zoo](https://github.com/openvinotoolkit/open_model_zoo/).\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Imports"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "collapsed": false,
+    "jupyter": {
+     "outputs_hidden": false
+    },
+    "pycharm": {
+     "name": "#%%\n"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "import librosa\n",
+    "import librosa.display\n",
+    "import IPython.display as ipd\n",
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import scipy\n",
+    "from openvino.inference_engine import IECore\n",
+    "from os import path"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Settings\n",
+    "\n",
+    "In this part you have to set up all variables further used in notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model_folder = \"model\"\n",
+    "download_folder = \"output\"\n",
+    "data_folder = \"data\"\n",
+    "\n",
+    "precision = \"FP16\"\n",
+    "model_name = \"quartznet-15x5-en\""
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Download and Convert Public Model\n",
+    "\n",
+    "We use `omz_downloader` and `omz_converter`, which are command-line tools from the `openvino-dev` package. \n",
+    "\n",
+    "If it is your first run, models will download and convert here. It might take up to ten minutes. "
```

Comment:
```suggestion
    "If it is your first run, models will be downloaded and converted here. It might take up to ten minutes. We use `omz_downloader` and `omz_converter`, which are command-line tools from the `openvino-dev` package. \n",
```